XenAdmin/Wizards/NewVMWizard/NewVMWizard.cs

```diff
@@ -221,7 +221,7 @@ protected override void UpdateWizardContent(XenTabPage senderPage)
 
                 RemovePage(pageVgpu);
                 gpuCapability = Helpers.GpuCapability(xenConnection) && selectedTemplate.CanHaveGpu();
-                if (gpuCapability)
+                if (gpuCapability && Helpers.GpusAvailable(xenConnection))

```

Comment:
I think the check Helpers.GpusAvailable(xenConnection) should move to the assignment of gpuCapability above, because it is used in the Finish() method of this wizard to decide wether we're passing vGpus in the creation action.